core/server/master/src/main/java/alluxio/master/metrics/MetricsStore.java

```diff
@@ -209,7 +223,7 @@ public void clear() {
       for (Counter counter : mClusterCounters.values()) {
         counter.dec(counter.getCount());
       }
-      mLastClearTime = System.currentTimeMillis();
+      mLastClearTime = mClock.millis();
```

Comment:
what's the advantage to using `mClock` vs `System.currentTimeMillis`?

Reply:
They are nearly the same, but mClock is good for testing, I can manually modify the time in unit tests instead of waiting for time to passed.